packages/@aws-cdk/aws-eks/test/integ.eks-cluster.expected.json

```diff
@@ -2231,7 +2231,7 @@
               },
               "/",
               {
-                "Ref": "AssetParametersc6510ec9618c99417a04af8cc6ed10b424085b76e94a020e26ae0b5cf3178cf6S3Bucket41E299BC"
```

Comment:
Do we know why those changed? Can we avoid or at least verify that the actual template is exactly the same?